soccer/planning/planner/CollectPlanner.cpp

```diff
@@ -0,0 +1,470 @@
+#include "CollectPlanner.hpp"
+
```

Comment:
This is almost line-for-line the same as the original collect planner. There should be no major functional changes.